requirements/requirements_dev.txt

```diff
@@ -1,7 +1,10 @@
 ansible  # Used in build process to generate some configs
 build
-django-extensions
+Django>=4.2.5,<4.3.0
```

Comment:
what change introduces this version constraint?

Reply:
this what the project pins to as whole, we just didn't have django in the dev environment so its not picked up in the bootstrapping of a dev environment so I added it to the dev reqs since in some cases (needing any kind of django management utility for example), django is needed in the virtenv.